compiler/Eta/BasicTypes/Id.hs

```diff
@@ -482,7 +482,7 @@ isImplicitId id
         PrimOpId {}      -> True
         DataConWorkId {} -> True
         DataConWrapId {} -> True
-                -- These are are implied by their type or class decl;
+                -- These are implied by their type or class decl;
```

Comment:
Doubled words are _usually_ typos and my tooling flags them and I try to correct them by hand. There are certain cases where the algorithm is wrong. Generally `that that` is acceptable/intended. Here, I don't think the doubling is intended, and hence I've removed it.